style.css

```diff
@@ -251,12 +257,13 @@ body {
 .about-section {
   width: 100%;
   height: auto;
-  padding-bottom: 150px;
+  padding-bottom: 100px;
   padding-top: 100px;
   background-image: url(img/about-img/about\ bg.png);
   background-repeat: no-repeat;
   background-size: cover;
   border-top-right-radius: 80px 80px;
+  margin-top: 100px;
 }
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the "About" section :clap: However it would be nice to use `background-color` instead of image
**HIT: Use this CSS code for the `.about-section` on the mobile version**
```sh
.about-section {
    width: 100%;
    height: auto;
    padding-bottom: 100px;
    padding-top: 100px;
    border-top-right-radius: 130px;
    background-color: white;
}
```
### Your design :point_down: 
![Screenshot from 2023-05-06 12-43-42](https://user-images.githubusercontent.com/91142432/236631624-1f6b298d-007b-4eae-a1a4-bcac743c1046.png)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-04-05 06-38-27](https://user-images.githubusercontent.com/91142432/236631672-950cb0a1-e40a-4f89-8446-55eb5a322166.png)

_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_